mne/gui/_coreg.py

```diff
@@ -794,18 +827,25 @@ def _fit_fiducials(self):
             verbose=self._verbose,
         )
         end = time.time()
-        self._renderer._status_bar_show_message(
+        self._display_message(
             f"Fitting fiducials finished in {end - start:.2f} seconds.")
         self._update_plot("sensors")
         self._update_parameters()
+        self._update_distance_estimation()
 
     def _fit_icp(self):
+        if not self._lock_fids:
+            self._display_message(
+                "Fitting is disabled, lock the fiducials first.")
+            return
```

Comment:
In the future we should properly disable in a UI way (button.setEnabled(False) in Qt parlance for example) the things that people are not supposed to use